kbs/src/api_server.rs

```diff
@@ -115,6 +123,11 @@ impl ApiServer {
                             .route(web::get().to(api))
                             .route(web::post().to(api)),
                     )
+                    .service(
+                        web::resource("/metrics")
+                            .route(web::get().to(prometheus_metrics))
```

Comment:
This function name is the same as the [struct member name](https://github.com/confidential-containers/trustee/pull/775/files). Could we rename it into something else like `prometheus_metrics_handler`?

Reply:
Sure, the  name was modelled after the `api()` handler name but I'm happy to rename the function.